stdlib/LibGit2/src/types.jl

```diff
@@ -45,6 +45,9 @@ Matches the [`git_time`](https://libgit2.github.com/libgit2/#HEAD/type/git_time)
 struct TimeStruct
     time::Int64     # time in seconds from epoch
     offset::Cint    # timezone offset in minutes
+    @static if LibGit2.VERSION >= v"0.27.0"
+        sign::Cchar
+    end
```

Comment:
Why not just drop the version check?

Reply:
I guess theoretically we support multiple libgit2 versions? But yes, we could certainly drop the checks and require 0.27

Reply:
Since the code divergence is small, I think it's better to be more compatible.  Building from-source should always grab the version we have in `deps/libgit2.version`, but distributions are probably quite a ways behind.

Reply:
Sounds good to me